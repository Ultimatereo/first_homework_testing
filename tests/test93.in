5
-269768697857 919992314453 199915018591 114735406393 -274599086692 197806280973
-955469393688 21172544604 -321875828979 -433786206397 64698751161 791140015472
-960124641999 -504280132781 -459704213082 -593989534909 -113629307289 -929706111109
-612791786118 645406098319 -62398223387 -552297940918 -283513762297 -568663929183
620433514473 520699541800 402348592623 233497145906 762244105461 843298694929
