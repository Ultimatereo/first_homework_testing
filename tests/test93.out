4.83025
6.64858
-21.8317
4.47636
2.78554
